build/pause/Makefile

```diff
@@ -12,20 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY:	build push
+CFLAGS = -O2 -Wall -static
 
-TAG = 2.0
+#TRIPLE = arm-linux-musleabi
+#IMAGE = docker.io/uluyol/pause-arm
+#TAG = 0.1
 
-ARCH = amd64
-IMAGE = pause
-# ARCH = arm
-# IMAGE = pause-arm
+TRIPLE = x86_64-linux-musl
+IMAGE = docker.io/uluyol/pause
+TAG = 0.1
```

Comment:
REPO and TAG need to be changed before submission along with docker push command.